### PR TITLE
javax.security.jacc + javax.security.enterprise + javax.security.auth => jakarta.security.*

### PR DESCRIPTION
**Pull request by scottmarlow**

### CODE REVIEW

File: src/com/sun/ts/tests/jaspic/tssv/module/servlet/TSServletWrapperSAM.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c)  2014, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c)  201, 2020 Oracle and/or its affiliates. All rights reserved.
```
Comment (alwin-joseph) [inline src/com/sun/ts/tests/jaspic/tssv/module/servlet/TSServletWrapperSAM.java:2]: typo.

File: bin/xml/impl/glassfish/common.xml [hunk 2]
```diff
@@ -1434,19 +1434,19 @@
     -->
     <target name="create.jacc.vm.options" depends="configPlatform">
          <antcall target="create-jvm-options" >
-          <param name="jvm.options" value=" -Djavax.security.jacc.policy.provider=com.sun.ts.tests.jacc.provider.TSPolicy"/>
+          <param name="jvm.options" value=" -Djakarta.security.jacc.policy.provider=com.sun.ts.tests.jacc.provider.TSPolicy"/>
```
Comment (alwin-joseph) [inline bin/xml/impl/glassfish/common.xml:1437]: Just for the record, these will create new failures during the setup but inevitable. These would be fixed by using the Glassfish  bundles with jakarta changes.

File: src/com/sun/ts/tests/jaspic/tssv/module/servlet/TSServletWrapperSAM.java (hunk 1 above)
Comment (scottmarlow) [reply src/com/sun/ts/tests/jaspic/tssv/module/servlet/TSServletWrapperSAM.java]: Thanks, resolved

File: bin/xml/impl/glassfish/common.xml (hunk 2 above)
Comment (scottmarlow) [reply bin/xml/impl/glassfish/common.xml:1437]: thanks @alwin-joseph!  I created [issue 239 for ](https://github.com/eclipse-ee4j/jakartaee-tck/issues/239) tracking this.

**Pull request merged**